product_docs/docs/migration_toolkit/55/installing/macos.mdx

```diff
@@ -67,7 +67,7 @@ If you are using PostgreSQL, you can invoke the graphical installer with Stack B
    After installing Migration Toolkit, you must install the appropriate JDBC drivers before performing a migration. See [Installing a JDBC driver](/migration_toolkit/latest/installing/installing_jdbc_driver/) for more information.
 
    !!! Note
-       The EDB JDBC driver is not available for Mac OS. 
+       The EDB JDBC driver is not available for macOS. 
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.